woocommerce/woocommerce-android
# Gift card issues

Closes: #8828


### Description
This PR solves different issues detected while running the Gift Cards testing plan:

1. Using a big font can cause the gift card code to overlap with the gift card's amount used

| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://user-images.githubusercontent.com/18119390/232123680-b8395942-76ce-46dd-8ee9-dc3c660ef61a.png" />  | <img width="300" src="https://user-images.githubusercontent.com/18119390/233106890-85c95ce9-a354-4548-a738-8e9cbfa26050.png">  |

Fixed in 741969bd182034056a5f1258477ef726503d0d00


2. Read the whole gift card info while using Talkback 

| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://user-images.githubusercontent.com/18119390/232124005-1bb9917a-7ca1-428a-b657-4bc1f82efd1c.png" />  |  <img width="300" alt="Screenshot 2023-04-19 at 08 31 32" src="https://user-images.githubusercontent.com/18119390/233108390-09f4d185-ff14-4b82-bac1-4a8d7a8012d1.png"> |

Fixed in cce61cfe70f39a3a9336141a3c869a50d5af59a2


### Testing instructions
TC1 
1. Go to the device settings 
2. Search for the Font Size setting
3. Select the max `Font size` and the max `Display size` settings
4. Get back to the Woo app
5. Open the orders tab
6. Select an order with a gift card applied/redeemed
7. Check that the card code doesn't overlap with the gift card's amount used

TC2
1. Go to the device settings 
2. Enable TalkBack
3. Get back to the Woo app
4. Open the orders tab
5. Select an order with a gift card applied/redeemed
6. On the Gift card section, tap on a gift card
7. Check that the whole row is selected and that the gift card's code and amount are read together
